modules/mockk-agent-android/build.gradle.kts

```diff
@@ -14,6 +14,7 @@ val mavenDescription: String by extra("${project.description}")
 
 @Suppress("UnstableApiUsage")
 android {
+    namespace = "io.mockk.proxy.android"
```

Comment:
Nit: is there a reason why this is not "io.mockk.agent.android" like the module name?

Reply:
Because the namespace of `mockk-agent-android-dispatcher` is `io.mockk.proxy.android.dispatcher`.
I aligned with it.
https://github.com/mockk/mockk/blob/ff79a07e3364b64fa1000d7967861d635f643395/modules/mockk-agent-android-dispatcher/build.gradle.kts#L11

Reply:
Should I match the `namespace` of both this module and the `mockk-agent-android-dispatcher` module to the name of the module?

Reply:
Nah, it's good as it is.

Thanks!